CMakeConfig.txt

```diff
@@ -34,6 +34,7 @@ set(PROTOCOL_ELRO_SWITCH ON CACHE BOOL "support for the Elro switch protocol")
 set(PROTOCOL_ELRO_CONTACT ON CACHE BOOL "support for the Elro contact protocol")
 set(PROTOCOL_EURODOMEST_SWITCH ON CACHE BOOL "support for the Eurodomest switch protocol")
 set(PROTOCOL_EV1527 ON CACHE BOOL "support for the EV1527 contact protocol")
+set(PROTOCOL_KERUI ON CACHE BOOL "support for the Kerui contact protocol")
```

Comment:
Should Kerui be in all capitals or not?

Reply:
fixed, at the sensor it is spelled in all capitals